src/shared/asset-selector/asset-list-item/asset-list-item.tsx

```diff
@@ -9,13 +9,15 @@ interface Props extends Pick<ListRowProps, 'style'> {
     asset: Asset;
     onClick: (asset: Asset) => void;
     selectedAsset: Asset;
+    balance: string;
 }
 
 export const AssetListItem: FC<Props> = ({
     style,
     asset,
     onClick,
-    selectedAsset
+    selectedAsset,
+    balance
```

Comment:
the same here. set that balance could be undefined in type and set a default value here